content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -625,174 +471,72 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
-$ curl --header "Authorization: Bearer $TOKEN" $INGRESS_HOST/ip -s -o /dev/null -w "%{http_code}\n"
-200
-{{< /text >}}
-
-### End-user authentication with mutual TLS
-
-End-user authentication and mutual TLS can be used together. Modify the policy above to define both mutual TLS and end-user JWT authentication:
-
-{{< text bash >}}
-$ cat <<EOF | kubectl apply -n foo -f -
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
-metadata:
-  name: "jwt-example"
-spec:
-  targets:
-  - name: httpbin
-  peers:
-  - mtls: {}
-  origins:
-  - jwt:
-      issuer: "testing@secure.istio.io"
-      jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
-  principalBinding: USE_ORIGIN
-EOF
-{{< /text >}}
-
-And add a destination rule:
-
-{{< text bash >}}
-$ kubectl apply -f - <<EOF
-apiVersion: "networking.istio.io/v1alpha3"
-kind: "DestinationRule"
-metadata:
-  name: "httpbin"
-  namespace: "foo"
-spec:
-  host: "httpbin.foo.svc.cluster.local"
-  trafficPolicy:
-    tls:
-      mode: ISTIO_MUTUAL
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+        notRequestPrincipals: ["*"]
+    to:
+    - operation:
+        paths: ["/headers"]
 EOF
 {{< /text >}}
 
-{{< tip >}}
-If you already enable mutual TLS mesh-wide or namespace-wide, the host `httpbin.foo` is already covered by the other destination rule.
-Therefore, you do not need adding this destination rule. On the other hand, you still need to add the `mtls` stanza to the authentication policy as the service-specific policy will override the mesh-wide (or namespace-wide) policy completely.
-{{< /tip >}}
-
-After these changes, traffic from Istio services, including ingress gateway, to `httpbin.foo` will use mutual TLS. The test command above will still work. Requests from Istio services directly to `httpbin.foo` also work, given the correct token:
-
-{{< text bash >}}
-$ TOKEN=$(curl {{< github_file >}}/security/tools/jwt/samples/demo.jwt -s)
-$ kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
-200
-{{< /text >}}
-
-However, requests from non-Istio services, which use plain-text will fail:
-
-{{< text bash >}}
-$ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..metadata.name}) -c sleep -n legacy -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
-000
-command terminated with exit code 56
-{{< /text >}}
-
 ### Cleanup part 3
 
 1. Remove authentication policy:
 
     {{< text bash >}}
-    $ kubectl -n foo delete policy jwt-example
-    {{< /text >}}
-
-1. Remove destination rule:
-
-    {{< text bash >}}
-    $ kubectl -n foo delete destinationrule httpbin
+    $ kubectl -n istio-system delete requestauthentication jwt-example
```

Comment:
Need to also remove the authorization policy.

Reply:
done